Presentation.Web/app/models/ui-customization/config/blueprints/it-contract-ui-customization-config-blueprint.ts

```diff
@@ -15,6 +15,42 @@
                     },
                     procurementPlan: {
                         text: "Genanskaffelsesplan"
+                    },
+                    contractId: {
+                        text: "KontraktID"
+                    },
+                    contractType: {
+                        text: "Kontrakttype"
+                    },
+                    purchaseForm: {
+                        text: "Indkøbsform"
+                    },
+                    extSigner: {
+                        text: "Leverandørs kontraktunderskriver"
+                    },
+                    extSigned: {
+                        text: "Leverandørs underskrevet"
+                    },
+                    extDate: {
+                        text: "Leverandørs dato"
+                    },
+                    intSigner: {
```

Comment:
no need for abbreviations.. ext = supplierSignature.. int = InternalSignature etc